frontend/src/app/components/janus/janusVideoRoom.service.ts

```diff
@@ -234,13 +234,9 @@ export class JanusVideoRoomService {
         return this.stopAndStartSdiForwarding(shidurState, user, forwardIp, videoPorts[index], audioPort, changeTitle);
       });
 
-      this.$q.all(forwardPromises).then(() => {
-        this.$log.info('VideoRoom - all remote feeds forwarded successfully');
+      (<any> this.$q).allSettled(forwardPromises).then(() => {
```

Comment:
This looks good for saving the Shidur State no matter what.

How are we dealing with errors in a specific call to stopAndStartSdiForwarding ?
Can we rollback somehow ? inform the user ? retry ?